plugins/extractors/postgres/postgres.go

```diff
@@ -181,9 +178,14 @@ func (e *Extractor) getTableMetadata(db *sql.DB, dbName string, tableName string
 		return result, nil
 	}
 
+	host, err := splitURL(e.config)
+	if err != nil {
+		err = errors.Wrap(err, "failed to split host from connection string")
+		return
+	}
 	result = &assetsv1beta1.Table{
 		Resource: &commonv1beta1.Resource{
-			Urn:     models.TableURN("postgres", e.config.Host, dbName, tableName),
```

Comment:
We could also still have a `host` config but instead of exposing it and populated externally, we populate it internally with parse url.